### ISSUE TITLE
Incorporate logic for Pay-To-Anchor Address types

### ISSUE THREAD

**Issue by Gudnessuche** [rust-bitcoin/rust-bitcoin#3702]
Given the existence of Topological Restricted Until Confirmed (TRUC) transaction in the latest version of Core, I say it'd be best we had logic implemented for P2A, just like P2TR got its.

**Comment by apoelstra** [comment]
Agreed. But right now our highest priorities are `units`, `hashes`, `primitives`, `io`. So it may be a little while before we can focus energy on `addresses`, which is where this stuff will live. (But PRs are welcome!)

**Comment by Gudnessuche** [comment]
@apoelstra any thoughts on when Silent Payments get implemented. Also have you looked at @josibake's PR to secp's library?

That thing's begging for a merge. Make it happen @apoelstra

**Comment by apoelstra** [comment]
Hah :) if only I had 2 more of me.

Josie's PR is on my short list. But SP in rust-bitcoin is also a "after we get primitives/hashes/etc out the door" thing.

**Comment by yancyribbens** [comment]
> Hah :) if only I had 2 more of me.

@apoelstra hasn't seen the movie Multiplicity I guess :)
